### PR TITLE
add the missing translations for pt-BR language in user fields and the missing key for 'Block all outgoing mail sent to this user.'

### PR DESCRIPTION
**Pull request by matheusrrocha**
1. Translate the 'groups' lang key in pt-br/lang.php
2. Add the two missing keys for 'create_password' and 'create_password_comment' in pt-br/lang.php
3. Add the missing translation in fields.yaml for the text 'Block all outgoing mail sent to this user.'
4. Add the key 'block' in en/lang.php
5. Add the key 'block' in pt-br/lang.php

### CODE REVIEW

**Comment by matheusrrocha** [comment]
Should i update the version.yaml?

**Comment by daftspunk** [comment]
That's fine, no need to update the version.yaml file, these changes will become part of the next version.

Thanks!

File: lang/en/lang.php [hunk 1]
```diff
@@ -96,7 +96,8 @@
         'empty_groups' => 'There are no user groups available.',
         'avatar' => 'Avatar',
         'details' => 'Details',
-        'account' => 'Account'
+        'account' => 'Account',
+        'block' => 'Block all outgoing mail sent to this user.'
```
Comment (daftspunk) [inline lang/en/lang.php]: This should be called `block_mail`

**Comment by matheusrrocha** [comment]
Done.

**Pull request merged**